bin/citrea/src/main.rs

```diff
@@ -206,9 +204,9 @@ where
         .runner
         .clone()
         .map(|runner| runner.sequencer_client_url);
-    let soft_confirmation_rx = match node_type {
+    let l2_block_rx = match node_type {
         NodeType::Sequencer(_) | NodeType::BatchProver(_) | NodeType::FullNode => {
-            soft_confirmation_channel.1
+            l2_block_channel.1
```

Comment:
let (l2_block_channel_rx, l2_block_channel_tx) = l2_block_channel;

and then use l2_block_channel_tx instead of l2_block_channel.1